app/src/main/res/layout/fragment_suggested_edits_add_descriptions_item.xml

```diff
@@ -52,6 +51,8 @@
                 android:id="@+id/headerView"
                 android:layout_width="match_parent"
                 android:layout_height="48dp"
+                android:paddingLeft="16dp"
```

Comment:
Would it be better if using `paddingStart` and `paddingEnd`?

Reply:
Done